### PR TITLE
WIP: Switch to doc theme

### PR DESCRIPTION
**Pull request by iluetkeb**

### CODE REVIEW

File: _data/authors.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+lui3si:
```
Comment (ralph-lange) [inline _data/authors.yml]: Why not your Github username iluetkeb?

File: _docs/FIROS2/index.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
 ---
-layout: default 
+title: micro-ROS & FIWARE
```
Comment (ralph-lange) [inline _docs/FIROS2/index.md]: microROS without hyphen?

File: _docs/index.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+---
+permalink: /docs/home/
+title: Overview
+---
+
+[micro-ROS](https://cordis.europa.eu/project/rcn/213167_en.html) puts ROS2 onto microcontrollers, making them first class participants of the ROS 2 environment.
```
Comment (ralph-lange) [inline _docs/index.md]: microROS without hyphen?

File: _docs/index.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+## Tutorials
+
+See [Tutorial List](/docs/tutorials/).
+
+## Architecture
+
+We're basically following the ROS2 architecture and make use of its middleware pluggability to use DDS-XRCE, which is suitable for micro-controllers. Moreover, we use an RTOS (NuttX) instead of Linux. On micro-controllers many different RTOS's are used, so we also add an RTOS abstraction layer (AL), to make porting to other RTOS possible.
```
Comment (ralph-lange) [inline _docs/index.md]: microcontroller in one word (https://en.wikipedia.org/wiki/Microcontroller)

Comment (ralph-lange) [inline _docs/index.md]: I propose to link DDS-XRCE with https://www.omg.org/spec/DDS-XRCE/

File: _docs/index.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+![](/img/micro-ROS_architecture.png)
+
+## Concept Documentation
+
+ - Library to support the implementation of language-specific ROS Client Libraries: [rcl](https://github.com/microROS/rcl)
```
Comment (ralph-lange) [inline _docs/index.md]: Given that rcl is used as is, I would removed this item (or, at least, move it to the end).

File: _docs/index.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+ - System modes and system hierarchy: [system_modes](system_modes/)
+ - Embedded transform (tf) library: [embedded_tf](embedded_tf/)
+ - Reference hardware: [https://github.com/microROS/hardware](https://github.com/microROS/hardware)
+ - Build infrastructure for embedded development using docker: [https://github.com/microROS/docker](https://github.com/microROS/docker)
+ - FIWARE interoperability: [FIROS2](FIROS2/)
+
+## Source Code Repositories
+
+ - Middleware
+  -  RMW layer [rmw-microxrcedds](https://github.com/microROS/rmw-microxrcedds)
```
Comment (ralph-lange) [inline _docs/index.md]: RMW adapter for Micro-XRCE-DDS.

Comment (iluetkeb) [reply _docs/index.md]: Changed

File: _docs/index.md (hunk 5 above)
Comment (iluetkeb) [reply _docs/index.md]: Removed

File: _docs/index.md (hunk 4 above)
Comment (iluetkeb) [reply _docs/index.md]: Linked

Comment (iluetkeb) [reply _docs/index.md]: hyphen removed.

File: _docs/index.md (hunk 3 above)
Comment (iluetkeb) [reply _docs/index.md]: Removed hyphen

File: _docs/FIROS2/index.md (hunk 2 above)
Comment (iluetkeb) [reply _docs/FIROS2/index.md]: Removed hyphen

File: _data/authors.yml (hunk 1 above)
Comment (iluetkeb) [reply _data/authors.yml]: Right.

File: _posts/2018-12-14-ric-eu.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
+---
+title: microROS@ROS Industrial 2018
+author: lui3si
```
Comment (ralph-lange) [inline _posts/2018-12-14-ric-eu.md]: There are several more lui3si to replace with iluetkeb.

**Comment by imuguruza** [comment]
I like the new theme, but I haven't thoroughly gone through the content.

**Comment by imuguruza** [comment]
@BorjaOuterelo what opinion do you have?

**Pull request merged**